crates/loader/src/local/mod.rs

```diff
@@ -60,6 +64,7 @@ pub async fn raw_manifest_from_file(app: &impl AsRef<Path>) -> Result<RawAppMani
         .with_context(|| anyhow!("Cannot read manifest file from {:?}", app.as_ref()))?;
 
     let manifest: RawAppManifestAnyVersion = toml::from_slice(&buf)?;
+    validate_raw_app_manifest(&manifest)?;
```

Comment:
Placing this here will cause it to run on `build`, which normally only cares about compilation... I'm not sure of a good solution to this.  We could place this call in the `from_file` and `from_bindle` wrappers used by `up`, and call it manually from `deploy`.  But then it starts to feel a bit like whack-a-mole...

Reply:
Yeah, I agree with it. We should have a place to transfer `RawAppManifest` to `Application.` Also, other functions/commands should rely on the `Application` object, so we don't need to put the validating function everywhere.